srml/sudo/src/lib.rs

```diff
@@ -116,6 +117,7 @@ decl_module! {
 		/// - Limited storage reads.
 		/// - No DB writes.
 		/// # </weight>
+		#[weight = SimpleDispatchInfo::FixedOperational(1_000)]
```

Comment:
Interestingly, I got this number by averaging all other functions that were benchmarked.